language/compiler/src/main.rs

```diff
@@ -72,6 +72,7 @@ fn do_verify_module(module: CompiledModule, dependencies: &[VerifiedModule]) ->
 }
 
 fn write_output(path: &PathBuf, buf: &[u8]) {
+    // TODO: panic is not option here. Provide feedback via println! and fail gracefully.
```

Comment:
perhaps @sblackshear or somebody else from language should look at these TODOs

Reply:
If you create an issue and assign it to Sam, that should do the trick.